src/generator.py

```diff
@@ -151,9 +150,13 @@ def convert_to_ogg(track_file, internal_name, create_tmp=True, cleanup_tmp=False
     if not os.path.isfile(tmp_track):
         return Status.BAD_MP3_META
 
-    #convert file
-    ffmpeg.convert(tmp_track, out_track)
-    #ffmpeg.options("-i %s -c:a libvorbis -o %s" % (tmp_track, out_track) )
+    if mix_mono:
+        #convert file in Mono
+        ffmpeg.convert(tmp_track +" -ac 1", out_track)
```

Comment:
Is this syntax officially supported by `pyffmpeg`, like is there documentation for it? I couldn't find anything (though it definitely works correctly)

Reply:
Sadly not